Bug report changes (#7709)
koreader/koreader: .github/ISSUE_TEMPLATE/bug_report.md

* `.adds/koreader/` directory for Kobo
* `applications/koreader/` directory for Pocketbook

> Android logs are kept on memory. Please go to Help->Bug Report to dump these logs to a file

Reviewer: ```suggestion
Android logs are kept in memory. Please go to Help → Bug Report to save these logs to a file.
```
